### DISCUSSION TITLE
Discord video/audio call to have a community Q&A session

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#661] [Ideas] [answered]
The community suggested having a Discord call, to learn from each other and practice our communication. I think we should try it, not sure how it will work with so many people, but we can try 🤓 

Ideas on how it could work:

- Use a GitHub Discussion where people can write a question that could then be asked during the call
- We go down the Discussion and the author of that thread can ask the question and we can discuss
- People can also write replies to the thread in the Discussion
- Discussion is a great way to document the Discord call
- New Discord voice channel and role for people who sign up (I have created a signup page, this will allow me to keep everyone up to date on when / what are happening)

Making notes here for rules:

- People must read [Code of Conduct](https://github.com/EddieJaoudeCommunity/support/blob/main/CODE_OF_CONDUCT.md)
- Stay on mute until talking
- Do not interrupt anyone
- Call might be recorded and possibly shared on socials

Topics people can talk about:
- Practice public speaking and communication 
- Give yourself a shoutout and share recent achievements
- What challenges do you have 
- Questions to the community
- What are your goals for 2021

Collab
- EquiCode
- Developer Student Community
- DevScript

Any thoughts suggestions please leave below...

**Comment by Vyvy-vi** [comment]
In future, we could also consider Google Meet(I think there are no video limits there)

We could perhaps have a moderator/manager to give the video perms, when a participant is specifically talking(when their discussion is being talked about)

Q) Why?
Ans- discord's voice calls can have 500+ members, but I think more than 25 ppl with their video on at the same time, is the limit of video participants.

Q) How can we do this?
Ans- Make a secondary `voice` role and we could give that video permissions, and disable video for everyone else...


The signup page is a great idea :+1:
Does it automatically add the vc role, when someone signs up?
(If yes, can you share how 🤓 )

**Reply by eddiejaoude** [reply]
> In future, we could also consider Google Meet(I think there are no video limits there)

I think Google Meet has a limit too, of ~12 people, but for paid accounts it might be 100 (I do have a paid account).

> We could perhaps have a moderator/manager to give the video perms, when a participant is specifically talking(when their discussion is being talked about)

We will need mods as it grows, for example if some is continually interrupting and not respecting other people, they will need to go.

> Q) Why?
Ans- discord's voice calls can have 500+ members, but I think more than 25 ppl with their video on at the same time, is the limit of video participants.
Q) How can we do this?
Ans- Make a secondary voice role and we could give that video permissions, and disable video for everyone else...

Oh interesting, if audio only it can have more people - great suggestion that is something we will need to look in to. Hopefully not for the first "event", as I would like to keep it as simple as possible to begin with and see how it goes. Then we can improve as we go along.

> Does it automatically add the vc role, when someone signs up?

Hmm I do not think it will add the role automatically 🤔 - we might be able to do it with a GitHub Action in the future though.

**Reply by eddiejaoude** [reply]
Google Meet worked well, but most people didn't have video on. This time we will try Discord with audio only, which has a limit of 5k members - audio I think might be more inclusive too

**Comment by adityaraute** [comment]
To be honest, I am not sure if I follow the idea behind GitHub Discussion. What is that gonna achieve exactly?
Is it gonna be before, after, or during the call?

**Reply by Vyvy-vi** [reply]
I think it is a way to schedule questions...
We could do that on discord too, BUT doing it on github makes it easier to document and would ensure that there is less off-topic chat

**Reply by eddiejaoude** [reply]
yep, I was going to use the GitHub Discussion as an order of questions / topics to go through and discuss - so I would go down the list. Then if someone wrote "what advice for ...", they would still speak it out and explain more on the call, and people can give feedback suggestions on GitHub Discussion thread also so it is not lost after the call - it documents the feedback from the community.

It probably needs tweaking after we try, but I thought it would be a good starting point. I am open to suggestions / tips etc

**Reply by adityaraute** [reply]
Okay this makes sense. It will be great way to compile the questions ~~unless we get too many in which case things can get messy~~.

Maybe we can begin with asking people to only ask a question once, and disable comments until the video call starts so that we the questions don't get answered already 😂 I mean it's not that bad, but the people on call might tend to go past questions that we think are already answered but perhaps aren't.

**Comment by mikeysan** [comment]
I'm sorry folks. Looking at this "The community suggested for a Discord community discussion" I have no idea what this topic is about which makes it difficult to follow the rest of the conversation.

Could someone please clarify?

**Reply by eddiejaoude** [reply]
@mikeysan you are right, I re-read what I wrote and it made no sense, unless people knew about it already 🤦‍♂️ . I have updated the title and a bit of the description. I hope now it makes more sense. If not let me know and I will update it again, as I think it needs to make sense for people to land on this discussion page.

**Comment by mikeysan** [comment]
That's great @eddiejaoude it makes a lot more sense now. Thank you. While reading through your changes I discovered that I could have edited it myself too, so I made some minor changes of my own. 

Please adjust or revert of my change isn't clear.

**Comment by eddiejaoude** [comment]
The **EddieHub** guild call went so well, I think this idea could work really well 👍  (we used Google Meet, limit of 100 people)

**Comment by mikeysan** [comment]
I believe we can also close this issue now since we figured out how to set up the calls and have had several successful ones now.

What do we think? Or do Discussions never end?

**Reply by eddiejaoude** [reply]
Good idea 👍 . We will tweak and make improvements as we go along also. Next one is in 16 hours 🎉

**Comment by adityaraute** [comment]
I think this discussion can serve as a notification for people. Or we can have a new discussion for it, since a lot of people keep missing the links or the timings.

**Reply by eddiejaoude** [reply]
I think a new discussion for each call would be good 👍 . I have created a mailing list to keep people notified. We might need to do other things too, I am open for suggestions 🤓

**Comment by eddiejaoude** [comment, marked as answer]
Everyone sign up to keep up to date with when we have community calls https://www.eddiehub.org/community-call-sign-up

**Reply by Nitesh-thapliyal** [reply]
It is amazing getting to know about folks, we should often do it🙌

**Reply by Vyvy-vi** [reply]
We should do this more often : )
It was great. Moreover the small timezone might be compensated, if we keep multiple scheduled events on discord for this :+1:

**Reply by adityaraute** [reply]
Yeah, nobody should have to stay awake after midnight or before 6am. Any time that fits this constraint?
